android-base.json

```diff
@@ -49,6 +49,9 @@
       "groupName": "com.fasterxml.jackson.*",
       "matchPackagePrefixes": [
         "com.fasterxml.jackson."
+      ],
+      "matchUpdateTypes": [
+        "patch"
```

Comment:
Have you tried (in any way) setting this in our repo's `renovate.json` file?
This is a public repo for general usage and the fact that we only want _patch updates_ is an implementation detail of our own repo. So, ideally, we should do it there, extending this configuration. Have you considered that @siarhei-kazlou ?

Reply:
Oh! I didn't realize it was a public repo. 😱

![breath](https://media.tenor.com/EEkiGqwvDRsAAAAM/sheldon-tbbt.gif)